partiql-planner/src/main/kotlin/org/partiql/planner/internal/typer/TypeLattice.kt

```diff
@@ -266,41 +266,57 @@ internal class TypeLattice private constructor(
                 SYMBOL to explicit(),
             )
             graph[DECIMAL] = relationships(
-                INT8 to explicit(),
-                INT16 to explicit(),
-                INT32 to explicit(),
-                INT64 to explicit(),
                 BOOL to explicit(),
-                DECIMAL to explicit(),
-                DECIMAL_ARBITRARY to explicit(),
+                INT8 to unsafe(),
+                INT16 to unsafe(),
+                INT32 to unsafe(),
+                INT64 to unsafe(),
+                INT to unsafe(),
```

Comment:
I agree those should be unsafe, but let's hold off for now to avoid behavior change.

Reply:
Either way we have a behavior change. These did not exist before.